### PR TITLE
Update owners (replace dgquintas and mehrdada)

### PR DESCRIPTION
**Pull request by jtattermusch**
Suggesting replacements for @dgquintas  and @mehrdada

### CODE REVIEW

**Comment by jtattermusch** [comment]
CC @apolcyn and @ncteisen

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

File: src/core/ext/filters/client_channel/OWNERS [hunk 1]
```diff
@@ -1,4 +1,4 @@
 set noparent
 @markdroth
-@dgquintas
+@ncteisen
```
Comment (markdroth) [inline src/core/ext/filters/client_channel/OWNERS]: Alex would probably be a better choice here.  Given his work on the c-ares resolver, he probably works on the client channel code more than Noah does.

Comment (jtattermusch) [reply src/core/ext/filters/client_channel/OWNERS]: Done.

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
Objective-C binary sizes
*****************STATIC******************
  New size                      Old size
 1,965,918      Total (=)      1,965,918

 No significant differences in binary sizes

***************FRAMEWORKS****************
  New size                      Old size
10,891,937      Total (<)     10,891,941

 No significant differences in binary sizes


```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by jtattermusch** [comment]
Known failures:
https://github.com/grpc/grpc/issues/16818
https://github.com/grpc/grpc/issues/15599
https://github.com/grpc/grpc/issues/15751

**Pull request merged**